clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -105,6 +105,24 @@ public class ConsumerConfig extends AbstractConfig {
     public static final String HEARTBEAT_INTERVAL_MS_CONFIG = CommonClientConfigs.HEARTBEAT_INTERVAL_MS_CONFIG;
     private static final String HEARTBEAT_INTERVAL_MS_DOC = CommonClientConfigs.HEARTBEAT_INTERVAL_MS_DOC;
 
+    /**
+     * <code>group.protocol</code>
+     */
+    public static final String GROUP_PROTOCOL_CONFIG = "group.protocol";
+    public static final String DEFAULT_GROUP_PROTOCOL = "generic";
+    public static final String GROUP_PROTOCOL_DOC = "The rebalance protocol consumer should use.  We currently " +
+        "support GENERIC or CONSUMER. If CONSUMER is specified, then the consumer group protocol will be used.  " +
+        "Otherwise, the generic group protocol will be used.";
+
+    /**
+    * <code>group.remote.assignor</code>
+    */
+    public static final String REMOTE_ASSIGNOR_CONFIG = "group.remote.assignor";
+    public static final String DEFAULT_REMOTE_ASSIGNOR = null;
+    public static final String REMOTE_ASSIGNOR_DOC = "The server side assignor to use. It cannot be used in " +
+        "conjunction with <code>group.local.assignor</code>. The group coordinator will choose the assignor if no " +
```

Comment:
Two questions:

1. Is `group.local.assignor` the same as `group.local.assignors`?
2. If `group.local.assignors` isn't in this PR, should we just omit that sentence?

Reply:
Good call. Yes.